integrated_channels/canvas/admin/__init__.py

```diff
@@ -0,0 +1,41 @@
+"""
+Canvas admin integration for configuring Canvas app to communicate with Canvas systems.
```

Comment:
maybe clean up the 'Canvas' redundancy in wording here? I don't mind it as it either though

Reply:
Hmmm the first one is probably the best removed, but otherwise meh.

Reply:
`Admin integration for configuring Canvas app to communicate with Canvas systems.` ? There is technically the Canvas system and app...